src/Lykke.Service.Bitcoin.Api/Controllers/OperationsController.cs

```diff
@@ -234,6 +239,11 @@ public async Task<IActionResult> BroadcastTransaction([FromBody] BroadcastTransa
         {
             if (request == null) throw new ValidationApiException("Unable deserialize request");
 
+            if (_settings.OperationsToForceRebuild?.Contains(request.OperationId) == true)
```

Comment:
i don't like if( bool? == true), better (bool? ?? false) i think. But let's ignore that

Reply:
Yep it's a matter of taste